package.json

```diff
@@ -21,61 +35,42 @@
     }
   ],
   "files": [
-    "dist/",
-    "src/",
-    "postinstall.js"
+    "src/"
```

Comment:
> You really want to limit the entry points to the module to prevent misuse. You don't want people importing from cross-domain-utils/src because they don't know what the underlying code is and how it needs to be compiled.

The expectation is, "if you import from src/, you're on your own" -- the recommendation is, you import from dist/, but if you know what you're doing and you want some of the extra granular benefits of getting the code directly from src/, that's still an option.

> I don't think minification is really an issue because when you import from cross-domain-utils you will get /dist/module/**/* which is the compiled source. The library that is using CDU as a dependency would do the minification which would minify it all.

Here's a use-case. The library gets included on https://cdnjs.com/. They use npm to get the latest version, and they add dist/*.js to the CDN. 

Here's another: I don't use webpack, but I do use npm to manage my dependencies. So I create an express route to point to dist/cross-domain-utils.min.js

Really don't see how it possibly hurts us to include all of dist/.

> You don't want to allow import from cross-domain-utils/dist because things may change such as file names.

Ideally the file names should not change unless with a major bump. I'd consider them part of the public interface of the module.

> Once all the libraries are not importing from src/ anymore 

I haven't yet heard a good answer for how this is going to work for:

- flow-runtime, which I don't think is going to work when importing from dist/module, since it type checks well beyond just the public interface of a module.
- feature flags like `__IE_POPUP_SUPPORT__` -- are we going to require 3rd parties to declare all of the feature flags we expect, in order to import from `dist/module`..?

> we need to remove the files array entirely so that main and module are the only two entry points. This will prevent misuse of importing and prevent headaches in the future.

How will `dist/module` be included in the npm package?